src/puppetbridge.ts

```diff
@@ -513,7 +563,12 @@ export class PuppetBridge extends EventEmitter {
 
 		// check if this is a valid room at all
 		const room = await this.namespaceHandler.createRoom(roomData);
-		if (!room || room.isDirect) {
+		if (room) {
```

Comment:
```suggestion
		if (!room) {
			return;
		}
		this.metrics.room.inc({type: room.isDirect ? "dm" : "group", protocol: this.protocol?.id});
```

Reply:
see https://github.com/Sorunome/mx-puppet-bridge/pull/72/files#diff-4cd484140d4c36875ceef0af75412976a1090961b49b5137435995661f73cb44R568 - unfortunately, githab has no "yes this is in the commit now"-detection